{
  "id": "BIT-harbor-2022-31667",
  "details": "Harbor fails to validate the user permissions when updating a robot account that\u00a0belongs to a project that the authenticated user doesn\u2019t have access to.\u00a0By sending a request that attempts to update a robot account, and specifying a robot\u00a0account id and robot account name that belongs to a different project that the user\u00a0doesn\u2019t have access to, it was possible to revoke the robot account permissions.",
  "aliases": [
    "CVE-2022-31667",
    "GHSA-xx9w-464f-7h6f"
  ],
  "modified": "2024-11-20T07:57:10.971119Z",
  "published": "2024-11-20T07:11:07.078Z",
  "database_specific": {
    "cpes": [
      "cpe:2.3:a:linuxfoundation:harbor:*:*:*:*:*:*:*:*"
    ],
    "severity": "Medium"
  },
  "references": [
    {
      "type": "WEB",
      "url": "https://github.com/goharbor/harbor/security/advisories/GHSA-xx9w-464f-7h6f"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "harbor",
        "ecosystem": "Bitnami",
        "purl": "pkg:bitnami/harbor"
      },
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "2.0.0"
            },
            {
              "fixed": "2.4.3"
            },
            {
              "introduced": "2.5.0"
            },
            {
              "fixed": "2.5.2"
            }
          ]
        }
      ],
      "database_specific": {
        "source": "https://github.com/bitnami/vulndb/tree/main/data/harbor/BIT-harbor-2022-31667.json"
      },
      "severity": [
        {
          "type": "CVSS_V3",
          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:C/C:N/I:L/A:L"
        }
      ]
    }
  ],
  "schema_version": "1.6.0"
}